mal/commands.py

```diff
@@ -70,3 +70,17 @@ def add(mal, args):
 def config(mal, args):
     # Show the current config file
     setup.print_config()
+
+
+def edit(mal, args):
+    """Edit an entry in the users list, if it was present. Notify otherwise."""
+    changes = dict()
+    for field in ['score', 'status', 'tags', 'add_tags']:
+        attr = getattr(args, field)
+        if attr is not None: changes[field] = attr
```

Comment:
PEP8, break the line.

Reply:
I always forget about this ahaha. Changing it now

Reply:
xD ok, it happens